### ISSUE TITLE
Question/Suggestion: Decentralizing TRON's core technology components

### ISSUE THREAD

**Issue by singh-abhijeet** [tronprotocol/java-tron#110]
I was going through [TRON's build guide](http://wiki.tron.network/en/latest/quick_start.html#download-and-build) and noticed TRON is utilizing Kafka (topics: block, transaction). Following are the queries related to this implementation:

1. Kafka being a centralized (broker based) pub-sub system, how will this fit in TRON's *decentralized Web 4.0* vision?
2. If we move forward with Kafka based implementation, who will control/administer this cluster? (TRON Foundation?)
3. Is TRON Dev Team planning to implement a masterless cluster management protocol for the Kafka cluster (implementing Paxos in Zookeeper)?

**TL;DR: How will TRON decentralize Web and entertainment ecosystem by utilizing centralized pub-sub messaging technology?**

**Comment by Rovak** [comment]
The Kafka system is part of Phase 1 of the three-step strategy (https://o836fhe91.qnssl.com/tron/whitebook/TronWhitepaper_en.pdf -> Consensus).

Kafka has already been deprecated in favor of Phase 3 which is under development right now. The Kafka guide should be removed from the documentation.

**Comment by singh-abhijeet** [comment]
Thanks @Rovak for the clarification.
